homeassistant/components/homeassistant_sky_connect/__init__.py

```diff
@@ -8,11 +8,19 @@
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.core import HomeAssistant
 
+from .util import create_entry_title
+
 _LOGGER = logging.getLogger(__name__)
 
 
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up a Home Assistant SkyConnect config entry."""
+
+    # Make sure the device config entry has the appropriate title suffix
+    expected_title = create_entry_title(entry)
+    if entry.title != expected_title:
+        hass.config_entries.async_update_entry(entry, title=expected_title)
```

Comment:
The user can change the title, and we should not override it.

If you want to align the old naming style with the new one, we should update the title during a config entry migration only if the user hasn't renamed it.
Increasing the minor version is enough for it

Reply:
I forgot to make this comment here as well: there is no way for a user to edit the config entry titles at the moment.